### PR TITLE
RTC7

### PR DESCRIPTION
**Pull request by ricardopereira**
Test for Suspended timeout.
Waiting some answer on #41.

### CODE REVIEW

File: ably-ios/ARTDefault.h [hunk 1]
```diff
@@ -14,8 +14,19 @@
...
-+ (NSTimeInterval)connectTimeout;
-+ (NSTimeInterval)disconnectTimeout;
-+ (NSTimeInterval)suspendTimeout;
+
++ (NSTimeInterval)connectTimeout; //Realtime
+
+/**
+ When the client is in the DISCONNECTED state, once this TTL has passed, the client should change the state to the SUSPENDED state signifying that the state is now lost i.e. channels need to be reattached manually.
+ 
+ Note that this default is override by any connectionStateTtl specified in the ConnectionDetails of the CONNECTED ProtocolMessage.
```
Comment (mattheworiordan) [inline ably-ios/ARTDefault.h]: Yes, this is true, but not yet implemented in any client libraries so does not need to be implemented

File: ablySpec/RealtimeClient.swift [hunk 2]
```diff
@@ -240,6 +240,48 @@ class RealtimeClient: QuickSpec {
...
+                            start = NSDate()
+                        default:
+                            break
+                        }
+                    }
+                }
+                client.close()
+
+                if let secs = endInterval {
+                    expect(secs).to(beLessThanOrEqualTo(UInt(options.suspendedRetryTimeout)))
```
Comment (mattheworiordan) [inline ablySpec/RealtimeClient.swift]: I am not sure this test is actually doing what is required?  The `suspendedRetryTimeout` represents the frequency of connection reattempts whilst in the disconnected state.  How exactly is this test checking this?

Comment (ricardopereira) [inline ablySpec/RealtimeClient.swift]: @mattheworiordan whilst in the disconnected state?! Did you mean suspended state?

Comment (ricardopereira) [inline ablySpec/RealtimeClient.swift]: @mattheworiordan the test is checking if the retry connection is occurring on the right time. The suspended retry timeout has been changed to 6 seconds and the client should respect that.

Comment (mattheworiordan) [inline ablySpec/RealtimeClient.swift]: Ah, yes, you are correct, thanks for clarifying

**Comment by mattheworiordan** [comment]
Can this be merged?

**Comment by ricardopereira** [comment]
@mattheworiordan Yes. Right now is just testing the `suspendTimeout`.

**Pull request merged**